### PR TITLE
Add VisualServoingVelocityExecutor class

### PR DESCRIPTION
**Pull request by egordon**
Adds a class built on top of the JacobianVelocityExecutor (See #605) to follow some arbitrary externally perceived transformation.

This class has been tested with Unit Tests only.

***

**Before creating a pull request**

- [x] Document new methods and classes
- [x] Format code with `make format`

**Before merging a pull request**

- [x] Set version target by selecting a milestone on the right side
- [ ] Summarize this change in `CHANGELOG.md`
- [x] Add unit test(s) for this change

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,12 @@
 * Control
 
   * Fixed bug in RosTrajectoryExecutor [#596](https://github.com/personalrobotics/aikido/pull/596)
+  * Add generic and position/velocity/effort Executor types [#602](https://github.com/personalrobotics/aikido/pull/602)
+  * Add JacobianExecutor wrapping any Velocity/EffortExecutor [#605](https://github.com/personalrobotics/aikido/pull/605)
```
Comment (RKJenamani) [inline CHANGELOG.md:9]: Remove both of these duplicates

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,9 +5,12 @@
...
   * Added generic and position/velocity/effort Executor types [#602](https://github.com/personalrobotics/aikido/pull/602)
   * Added JacobianExecutor wrapping any Velocity/EffortExecutor [#605](https://github.com/personalrobotics/aikido/pull/605)
   * Implemented generic Executors into Robot class [#614](https://github.com/personalrobotics/aikido/pull/614)
+  * Add VisualServoingVelocityExecutor wrapping JacobianExecutor [#620](https://github.com/personalrobotics/aikido/pull/620)
```
Comment (RKJenamani) [inline CHANGELOG.md:13]: Add -> Added

File: include/aikido/control/VisualServoingVelocityExecutor.hpp [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    double mGoalTolerance;
+    double mApproachVelocity;
+    std::chrono::duration<double> mTimeout;
+    std::chrono::milliseconds mThreadPeriod;
+  };
+
+  /// Constructor.
+  ///
+  /// \param eeNode End effector body node (current position =
+  /// getWorldTransform()) \param executor Base jacobian velocity executor.
```
Comment (RKJenamani) [inline include/aikido/control/VisualServoingVelocityExecutor.hpp:62]: \param executor should start on another line.

File: include/aikido/control/util.hpp [hunk 4]
```diff
@@ -47,6 +47,26 @@ inline T checkNull(T obj)
   return obj;
 }
 
+// TODO: Move below to a common util file
```
Comment (RKJenamani) [inline include/aikido/control/util.hpp:50]: @egordon Has this TODO been addressed?

**Review by RKJenamani — APPROVED** [review]

File: include/aikido/control/util.hpp (hunk 4 above)
Comment (egordon) [reply include/aikido/control/util.hpp:50]: Just addressed it.

**Pull request merged**